### PR TITLE
fix #59 - add a readme to the examples directory

### LINKED ISSUE #59: in the examples, libcdk5-dev needs to be installed
I think this is just a doc issue, but we should put a README in the ./examples directory.

analog@imhotep:~/github/libiio/examples$ make
cc -Wall   -c -o ad9361-iiostream.o ad9361-iiostream.c
cc -o ad9361-iiostream ad9361-iiostream.o  -liio
cc -Wall   -c -o dummy-iiostream.o dummy-iiostream.c
cc -o dummy-iiostream dummy-iiostream.o  -liio
cc -Wall   -c -o iio-monitor.o iio-monitor.c
iio-monitor.c:22:21: fatal error: cdk/cdk.h: No such file or directory
 #include <cdk/cdk.h>
                     ^
compilation terminated.

analog@imhotep:~/github/libiio/examples$ sudo apt-get install libcdk5-dev

fixes it

### PR DESCRIPTION
**Pull request by rgetz**
I will need to add signed-off-by before commit, but wanted to get people's opinion on what should be in this file.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## ad9371-iiostream
+  * Language : C
+
+This example libiio program is meant to exercise the features of IIO functionality on the AD9371.
+It takes the uri as the only argument. for example : `./ad9371-iiostream ip:192.168.2.1`
+
+## adrv9009-iiostream
+  * Language : C
+
+This example libiio program is meant to exercise the features of IIO functionality on the AD9371.
```
Comment (mhennerich) [inline examples/README.md]: typo -> ADRV9009

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# libiio Examples
+
+The libiio is a cross platform library for interfacing with Linux IIO devices. 
+
+These are some primative examples of using the libiio library.
```
Comment (dNechita) [inline examples/README.md]: primative - typo

**Comment by dNechita** [comment]
.gitignore needs to be update as well

**Comment by rgetz** [comment]
updated the Readme

**Pull request merged**